### PR TITLE
:sparkles: Add the ability to customize project name

### PR DESCRIPTION
**Pull request by tbetous**

### CODE REVIEW

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -2,10 +2,8 @@
   <header class="navbar" :class="{ offline: !networkOnLine }">
     <router-link to="/home">
       <img alt="logo-bento" class="logo" src="@/assets/img/bento-starter.svg" />
-      <span class="site-name">
-        Bento<span class="can-hide">Starter</span></span
-      ></router-link
-    >
+      <span class="site-name">{{ appTitle }}</span>
```
Comment (kefranabg) [inline src/components/NavBar.vue]: We need to check how it's displayed with "BentoStarter" value

File: src/views/Home.vue [hunk 2]
```diff
@@ -17,17 +17,19 @@
...
-    ]
+  head: function() {
+    return {
+      title: {
+        inner: 'Home'
+      },
+      meta: [
+        {
+          name: 'description',
+          content: `${this.appTitle} home page`,
```
Comment (kefranabg) [inline src/views/Home.vue]: Did you check that it's working correctly ?

File: internals/setup.js [hunk 3]
```diff
@@ -201,6 +223,15 @@ function printFail() {
   const requiredNpmVersion = npm.match(/([0-9.]+)/g)[0]
   await checkNpmVersion(requiredNpmVersion).catch(onError)
 
+  newProjectName = await askUserForNewProjectName()
+  if (newProjectName) {
```
Comment (kefranabg) [inline internals/setup.js]: What happen if projectName is empty ?

File: src/views/Home.vue (hunk 2 above)
Comment (tbetous) [reply src/views/Home.vue]: I checked what is generated when I build the app. In the dist I have the right head content. I checked also what is in the head content when I start a server. All is OK to me.

File: internals/setup.js (hunk 3 above)
Comment (tbetous) [reply internals/setup.js]: If no name is given then no file will be changed.

**Pull request merged**